Sources/Google/GoogleAuthenticator.swift

```diff
@@ -24,7 +24,9 @@ extension GoogleAuthenticator: Authenticator {
   /// SignIn user.
   ///
   /// Will return promise with the `Response` object on success or with `Error` on error.
-  public func signIn(from presentingViewController: UIViewController) -> Promise<Response> {
+  public func signIn(from presentingViewController: UIViewController,
+                     hint: String? = .none,
+                     scopes: [String]? = .none) -> Promise<Response> {
```

Comment:
What is a `hint` and what is `scopes`?

Reply:
**hint**
An optional hint for the authorization server, for example the user’s ID or email address, to be prefilled if possible.

**additionalScopes**
An optional array of scopes to request in addition to the basic profile scopes.

Reply:
when using GmailAPI for example, we need to request additional authentication scopes to manage user emails like fetching, reply etc.